shared/teams/team/invites/container.js

```diff
@@ -7,30 +7,6 @@ export type OwnProps = {
   teamname: string,
```

Comment:
Each tab that previously rendered a `List` (all but settings) now have connectors that output one prop: the array of list items. these connectors are delegated to in the main component to render the whole list flatly